### ISSUE TITLE
Unclear documentation

### ISSUE THREAD

**Issue by mathstuf** [handshake-org/hs-airdrop#24]
I'm not familiar with Handshake. I found this repo through a mention elsewhere and was curious. The docs seem to be making some assumptions.

Just looking at the command reference in the readme, it seems any SSH key call wants and address and amount. Is there an assumption that I already have some handshake account/wallet setup? Should that address be mine? How can I find out how much I can even request from the initial airdrop (the docs mention the initial amount, but if I transfer out some lower amount, how do I find out the balance)?

Basically, some more background on what is actually going on with each command and links to obtaining things like a valid address and amount request should be added.

**Comment by pinheadmz** [comment]
I agree we need better docs for users who have never used a cryptocurrency or blockchain before. I can start on that soon. You might want to read the wallets guide here: https://handshake-org.github.io/guides/wallet.html, and check out the api docs here: https://handshake-org.github.io/api-docs/

You should be able to run an `hsd` full node in `regtest` mode (local only, very easy to generate blocks and move send transactions between wallets on your own local machine). And you should be able to test your airdrop by creating a claim with `hs-airdrop` and submitting it to your regtest blockchain, using a regtest wallet address.

The amount you receive is determined by the number of aridrop participants and a few other factors. When you claim your airdrop, you do not set the amount -- you will receive the entire airdrop at the wallet address you provide. You do have the option to set a "fee" which is a blockchain concept in which the _sender_ pays a little money to the miners to include their transaction in a block. On regtest, you can probably set it to `0` or maybe `100` or so.

I'll get back to you with a more step-by-step guide. In the meantime see what you can learn on your own if you the time! I also have a much more in-depth article about how the airdrop works if you want to get in to the weeds: https://medium.com/@matthewzipkin/understanding-the-handshake-airdrop-and-reserved-names-428d9e90b560

**Comment by pinheadmz** [comment]
See #14

**Comment by tynes** [comment]
@mathstuf There is a bit of an [inferential gap](https://wiki.lesswrong.com/wiki/Inferential_distance) after thinking about blockchain for awhile, its easy to gloss over details that are assumed to be common knowledge with new people entering the space. Since there is essentially a bounty on figuring it out, people will eventually produce more detailed guides. If you figure anything out, feel free to make it easier for the next person by documenting what confused you and your new understanding in an issue.

See the docs here: https://handshake-org.github.io/

**Comment by pinheadmz** [comment]
Mainnet has started and all the software and the readme has been updated:

https://github.com/handshake-org/hs-airdrop/commit/cd90f42cf806dc4c037b03db5548cd14e3a5852a

Has this issue been resolved? @mathstuf
